### ISSUE TITLE
test run failure

### ISSUE THREAD

**Issue by kazumaxneo** [metagenlab/zDB#14]
Hi, 

I have read the paper and find zDB to be a very useful pipeline for comparative genomics. 
I would like to use this tool, but I got an error in the last process in the test run (picuture).
The same error  occur in "--conda", "--docker", and the default singlularity. 
Is there any solution to avoid this?

enviroment
v1.1.1 installed with conda
python=3.11.3
nextflow version 22.10.6.5843

![スクリーンショット 2023-06-09 23 34 43](https://github.com/metagenlab/zDB/assets/20102961/e67449ca-a1e4-46ba-bb11-753b1f60c941)

**Comment by bkm** [comment]
Hello,
Thanks for reporting this issue!
Judging from the code snippet, this is an old issue that is fixed in the current code base. 

Could run zdb without any argument to check that it is indeed v1.1.1 that is installed (just run 'zdb') and show me the exact command you used to run the analysis pipeline?

Best regards,
Bastian

**Comment by kazumaxneo** [comment]
Hi Bastian.

Thank you for your response !
This is the result of  typing "conda list | grep zdb" 
![1](https://github.com/metagenlab/zDB/assets/20102961/65ad317b-1bf0-4f6d-89de-9069ddd5615d)
I think version 1.1.1, right?
I installed zdb with conda. 

The error occurred when I executed minimum command as explained in the repository.
$ wget https://github.com/metagenlab/zDB/raw/master/test_dataset.tar.gz
$ tar xvf test_dataset.tar.gz

#Then, typed just explained in the repo.
$ zdb run --input=input.csv --name=simple_run
=> Error occured after a few minutes

Sincerely.
Kazu

**Comment by bkm** [comment]
Hello,
So yes, you are using version 1.1.1, but the build is not the most recent one.
When did you install the tool ?

Best regards,
Bastian

**Comment by bkm** [comment]
Hello,
In the end, I guess it's simpler to create a new version. Please simply reinstall zdb with the same command you used the first time. It should now be version 1.1.2 and include the fix for the problem you noticed.

Please tell me if this solves your issue. Thanks for your patience!

Best regards,
Bastian

**Comment by kazumaxneo** [comment]
Hi Bastian.

Thanks !, v1.1.2 worked.
One concern, the step where I reported the error did not get a check mark that the nextflow process was complete, but anyway, no error message was shown and successfully finished.
![1](https://github.com/metagenlab/zDB/assets/20102961/424a91f1-8a98-4fb6-a117-9b4e3c95aebf)

And I was able to run the web server without any problems.
$ zdb webapp --name=simple_run --allowed_host=127.0.0.1 --port=8080
Thank you very much. 

Sincerely.
Kazu
